specs/release-automation/flow-for-automation.puml

```diff
@@ -0,0 +1,65 @@
+@startuml
+title Release Automation Process
+
+start
+group RA-1: Trigerring the Process
+    :User passes link to excel dictionary on OSF to use for the release;
+    :User passes OSF personal access token;
+    :User triggers release process;
+end group
+
+group RA-2: Creating the Release Files
+    :Retrieve Excel Dictionary;
+    :Parse metadata from files sheet;
+    :Construct file names, ending in ".csv" or ".xlsx";
+    :Populate file with information from appropriate sheet;
+    :Add optional headers;
+    :Store file in appropriate directory based on its upload location;
+end group
+
+group RA-3: Deploying the files to GitHub
+    :Check if previous release exists on GitHub in the appropriate directory;
+
+    if (No previous release on GitHub) then (yes)
```

Comment:
Try to avoid negatives in your if statements. For example, this one would be better read as, "Is there a previous release on GitHub?"

Reply:
Can you update all the other instances where this happens?